building/tracks/concepts.md

```diff
@@ -168,3 +181,35 @@ Links can also optionally have an `icon_url` field, which can be used to customi
   }
 ]
 ```
+
+---
+
+### File: `.meta/config.json`
+
+**Purpose:** Contains meta information on the exercise.
```

Comment:
Perhaps "on the CONCEPT"? Btw it's repeated in line 193.

Reply:
Good catch: https://github.com/exercism/docs/pull/106